chef/cookbooks/hyperv/recipes/config.rb

```diff
@@ -90,7 +90,7 @@
 num_vlans = neutron_server[:neutron][:num_vlans]
 vlan_end = [vlan_start + num_vlans - 1, 4094].min
 
-template "#{node[:openstack][:config]}\\neutron_hyperv_agent.conf" do
+template "#{node[:openstack][:config].gsub!(/\\/, "/")}/neutron_hyperv_agent.conf" do
```

Comment:
Please just ```gsub``` and not ```gsub!```, the latter one manipulates the node attribute but it should be just printed here.